osu.Game/Screens/SelectV2/Panel.cs

```diff
@@ -109,53 +107,30 @@ private void load(OverlayColourProvider colourProvider, OsuColour colours)
             InternalChild = TopLevelContent = new Container
             {
                 Masking = true,
-                CornerRadius = corner_radius,
+                CornerRadius = CORNER_RADIUS,
                 RelativeSizeAxes = Axes.Both,
-                X = corner_radius,
-                EdgeEffect = new EdgeEffectParameters
-                {
-                    Type = EdgeEffectType.Shadow,
-                    Hollow = true,
-                    Radius = 2,
-                },
+                X = CORNER_RADIUS,
                 Children = new[]
                 {
-                    backgroundBorder = new Box
-                    {
-                        RelativeSizeAxes = Axes.Both,
-                        Colour = Color4.Black,
-                    },
-                    backgroundLayerHorizontalPadding = new Container
+                    backgroundContainer = new Container
                     {
                         RelativeSizeAxes = Axes.Both,
-                        Child = new Container
-                        {
-                            RelativeSizeAxes = Axes.Both,
-                            Masking = true,
-                            CornerRadius = corner_radius,
-                            Children = new Drawable[]
-                            {
-                                backgroundGradient = new Box
-                                {
-                                    RelativeSizeAxes = Axes.Both,
-                                },
-                                backgroundContainer = new Container
-                                {
-                                    RelativeSizeAxes = Axes.Both,
-                                },
-                            }
-                        },
                     },
                     iconContainer = new Container
                     {
                         Anchor = Anchor.CentreLeft,
                         Origin = Anchor.CentreLeft,
                         AutoSizeAxes = Axes.Both,
                     },
-                    Content = new Container
+                    contentPaddingContainer = new Container
                     {
                         RelativeSizeAxes = Axes.Both,
-                        Padding = new MarginPadding { Right = PanelXOffset + corner_radius },
```

Comment:
I can't find any reason for this padding, and now that I'm including the beatmap background in content, it applies this much padding to the background from the right, which looks weird in the panel tests.

The closest explanation I can come up is that this was used for the right-side pill count in group panels, but they still render correctly without the padding.